builtin/logical/database/dbplugin/plugin.go

```diff
@@ -29,6 +29,13 @@ type Statements struct {
 	RenewStatements      string `json:"renew_statements" mapstructure:"renew_statements" structs:"renew_statements"`
 }
 
+// UsernameConfig is used to configure prefixes for the username to be
+// generated.
+type UsernameConfig struct {
```

Comment:
I see, so the role name (and unix timestamp) is now included as part of the username for more verbosity as to who and when the token was created?